etcdctl/util/help.go

```diff
@@ -14,7 +14,7 @@
 
 // copied from https://github.com/rkt/rkt/blob/master/rkt/help.go
 
-package cobrautl
+package util
```

Comment:
Hey @ah8ad3 - Thanks for raising this - In order to ensure this pr is not blocked on wider consensus I would suggest not moving from `pkg` and keep this pr to only adding examples.  If we want to move it imo we should do that as a separate pr.

Reply:
I'm pretty sure that this file is safe to move here, and we should do that, because the only use of this file is `etcdctl`, but i move it back. PTAL
I will raise another PR for that.